app/controllers/events_controller.rb

```diff
@@ -37,13 +37,19 @@ def update
 
     if @event.valid?
       @event.save
-      redirect_to action: :index, c: 'eu-migration'
+      redirect_to action: :index
     else
       formify_event(@event)
       render action: :new, status: 400
     end
   end
 
+  def destroy
+    @event = EDM::Event.find_by(uuid: params[:uuid])
+    @event.destroy!
```

Comment:
if this doesn't work, we should catch the error and notify the user somehow, either render a forbidden error or just re-render the index action with a flash notice?
now it throws an error: ' Cannot delete EDM::Event because of dependent 'edm_wasPresentAt_for'.'